packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/Drawing/tools/Root.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const STROKE_WIDTH = 1.5
+const SELECTED_STROKE_WIDTH = 2.5
+
+const DrawingToolRoot = ({ active, children, tool }) => {
+  const mainStyle = {
+    color: tool.color,
+    fill: 'transparent',
+    pointerEvents: 'none',
+    stroke: tool.color,
+    strokeWidth: active ? SELECTED_STROKE_WIDTH : STROKE_WIDTH
+  }
+
+  return (
+    <g {...mainStyle} tabIndex='-1'>
```

Comment:
I think delete mark functionality can start from a handler/callback here and go back to the DrawingContainer

Reply:
Maybe add `focusable` here too to allow keyboard interaction in browsers that follow the original SVG spec and ignore `tabindex`. I'm thinking Edge but my knowledge of SVG is patchy and could well be wrong. 

Links may be the only way to reliably and easily add keyboard interaction across browsers, since SVG doesn't have buttons. From memory, that's how keyboard interaction is handled in the SVG account stats ribbon on the Zooniverse home page.